packages/webapp/pages/posts/[id]/comments/new.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import { useRouter } from 'next/router';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import CommentInputPage from '@dailydotdev/shared/src/components/comments/CommentInputPage';
+
+interface Props {
+  id: string;
+}
+
+const NewCommentPage = ({ id }: Props): ReactElement => {
+  const router = useRouter();
+  const replyCommentId = router.query.replyTo as string;
+
+  return <CommentInputPage postId={id} replyCommentId={replyCommentId} />;
+};
```

Comment:
It would be styled like the mobile page, but that should be fine for now, since we have no links pointing to this page on desktop. Everything still works on desktop, it's just big - e.g. the comment edit box takes most of the page. I think this is completely fine for this version.

<img width="1077" alt="Screenshot 2024-03-25 at 15 32 56" src="https://github.com/dailydotdev/apps/assets/9974711/0e7d9b37-298b-4dd6-84c1-ee5b06d5d7b4">

Reply:
Maybe a simple client side redirect to post page (since we have Id) if you are on mobile just to avoid any confusion.

Reply:
We can scroll with `/posts/{postId}/#c-{commentId}`

Reply:
I think from the point of view of SEO optimizations, it's actually better for us to have a comment path that the indexer could crawl. So my vote would be to keep it simple and not introduce another redirect. 

But let me know if you prefer to add it.

Reply:
Comment new path does not give any SEO value so I don't think it should even be indexed, so add noindex and nofollow to head would work. We already do it with `/squads/new`. As for redirect let's not do it for now.

Reply:
Added noindex and nofollow 👍